r-package/grf/vignettes/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.html
+*.R
```

Comment:
For my context, why do we .gitignore `R` files here?

Reply:
It was the standard setup provided by `usethis::use_vignette`: generated files (Rmarkdown can produce both R and html) should not be added